src/typed-method-types/workflows/triggers/scheduled.ts

```diff
@@ -42,6 +44,18 @@ type BaseFrequencyType = {
   on_week_num?: 1 | 2 | 3 | 4 | -1;
 };
 
+type OnceFrequencyType = {
+  /** @description How often the trigger will activate */
+  type: typeof SCHEDULE_FREQUENCY.Once;
+};
+
+type HourlyFrequencyType =
+  & {
+    /** @description How often the trigger will activate */
+    type: typeof SCHEDULE_FREQUENCY.Hourly;
+  }
+  & Pick<BaseFrequencyType, "repeats_every">;
```

Comment:
Asked in channel, but also asking here — is there any limit around `repeats_every` on the API or will this accept any value?

Reply:
There aren't any limitation for repeats_every in API. Similarly, for monthly freq too we aren't restricting it to just 12 months only. User can set it to repeat after every 1 hr or 12hrs or 50hrs.